### DISCUSSION TITLE
Crysol - using the same fitting parameters

### DISCUSSION THREAD

**Discussion by franke-biosaxs** [biosaxs-com/atsas-community#1481] [Model evaluation and manipulation]
On 2016-06-13 16:02:13, user McBudaTea wrote:

Hello,

I want to calculate the SAXS profiles of a large number of different structures of the same Protein and compare them with the experimental data. Because of that, I would like to use the fitting parameters of a reference structure (e.g. the crystal structure).
According to the manual, Crysol uses three parameters for the fit. While it is possible to use the dro keyword to specify the contrast of the hydration shell, I could not find the keyword to fix the average displaced solvent volume per atomic group or the relative background.
How can it be done?

**Comment by franke-biosaxs** [comment]
On 2016-06-13 19:00:34, user Alex wrote:

i don't think there is a way to do it using known options, however, you can try following:

1. create an "ansX.txt" file (see how it is done in DAMMIF, for example).
In CRYSOL interactive mode there is "Another set of parameters?", where you can explicitly specify the parameters that you mentioned.
Maybe it is possible to have there min=max of parameter of interest (you have to check this).

2. write a small program that will based on your ansX.txt make ans_XXXX.txt file for each of your PDB file, where X is a digit.
Basically, it should just replace the name of the PDB file in your ansX.txt, so that there is an answer file for each PDB file.

3. run CRYSOL using all answer files.. smth like
$ \> for i in `ls ans*.txt`; do crysol \< $i; done

Good luck,
Alex
